src/main/resources/db/changelog/changes/v1.0.0/sql/2021-05-25-01__create_contribution_criteria_configuration_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE contribution_criteria_configuration
+(
+    central_server_id uuid NOT NULL,
+    created_by CHARACTER VARYING(255),
```

Comment:
`varchar` seems like a more widely used name for such kind of type. consider replacing with 'varchar'

Reply:
It was copypasted from database creation script generated by latest version of PgAdmin 4. If it is not in compliance with FOLIO mainstream it will be replaced to VARCHAR.